src/Uno.UI/UI/Xaml/UIElement.Pointers.Android.cs

```diff
@@ -43,125 +50,221 @@ partial class UIElement
 			}
 		}
 
-		protected sealed override bool OnNativeMotionEvent(MotionEvent nativeEvent, View originalSource, bool isInView)
+		protected sealed override bool OnNativeMotionEvent(MotionEvent nativeEvent)
```

Comment:
If I'm right currently there 2 cases where this method can be invoked now:
1. On the `VisualRoot` where you have to route the event to the `ICoreWIndowEvents` like you did here. In that case the `ownerEvents` will determine the `OriginalSource`, and start bubbling in managed code from this element. 
I would move this case to the [`dispatchTouchEvent`](https://developer.android.com/reference/android/view/View#dispatchTouchEvent(android.view.MotionEvent)) (and `dispatchGenericMotionEvent`) before Android even try to determine the target element.
3. When transitioning back from native bubbling to managed (i.e. when bubbling reached a native only element, we stopped managed bubbling and invoked something like `OnTouchEvent` on its parent to initiate / simulate native bubbling). 
In that case we must not forward to the `ownerEvents` but only resume managed bubbling (i.e. invoke `OnPointerXXX` methods).
Also, we should not re-create a new `PointerEventArgs` but re-use the already existing `PointerROUTEDEventsArgs` with the right `OriginalSource`.

Also I noticed that you remove the ability to raise multiple pointer events from a single `MotionEvent`. I think this is still relevant, especially for multi-touch (on Android a `MotionEvent` can contains info on multiple pointers at once while on Windows we will get invoked multiple times for each pointer).